apps/docs/content/components/Bento.mdx

```diff
@@ -0,0 +1,227 @@
+```
+
+### Example Layout 1
+
+```jsx live
+<Bento>
+  <Bento.Item columnSpan={12} rowSpan={2} />
+  <Bento.Item columnSpan={7} rowSpan={2} />
+  <Bento.Item columnSpan={5} rowSpan={2} />
+  <Bento.Item columnSpan={12} rowSpan={2} />
+</Bento>
+```
+
+### Example Layout 2
+
+```jsx live
+<Bento>
+  <Bento.Item columnSpan={12} rowSpan={2} />
+  <Bento.Item columnSpan={5} rowSpan={2} />
+  <Bento.Item columnSpan={7} rowSpan={2} />
+  <Bento.Item columnSpan={12} rowSpan={2} />
+</Bento>
+```
+
+### Example Layout 3
+
+```jsx live
+<Bento>
+  <Bento.Item columnSpan={7} rowSpan={2} />
+  <Bento.Item columnSpan={5} rowSpan={2} />
+  <Bento.Item columnSpan={12} rowSpan={2} />
+  <Bento.Item columnSpan={12} rowSpan={2} />
+</Bento>
+```
+
+### Dark Mode Items
+
+```jsx live
+<Bento>
+  <Bento.Item columnSpan={7} rowSpan={2} colorMode="dark" />
+  <Bento.Item columnSpan={5} rowSpan={2} colorMode="dark" />
+  <Bento.Item columnSpan={12} rowSpan={2} colorMode="dark" />
+  <Bento.Item columnSpan={12} rowSpan={2} colorMode="dark" />
+</Bento>
+```
+
+### Responsive behavior
+
+Use `columnSpan`, `rowSpan`, `columnStart`, `rowStart`, `flow`, `verticalAlign`, `horizontalAlign` with an `Object` of breakpoint-specific keys and values corresponding to each type to enable responsive behavior for each of these properties.
+
+Breakpoints use `min-width`, where it will also apply your chosen `span` value to all larger breakpoints.
+
+```jsx live
+<Bento>
+  <Bento.Item
+    columnSpan={{
+      xsmall: 12,
+      small: 12,
+      medium: 6,
+      large: 4,
+    }}
+    rowSpan={{xsmall: 1, small: 2, medium: 3}}
+  />
+  <Bento.Item
+    columnSpan={{
+      xsmall: 12,
+      small: 12,
+      medium: 6,
+      large: 4,
+    }}
+    rowSpan={{xsmall: 1, small: 2, medium: 3}}
+  />
+  <Bento.Item
+    columnSpan={{
+      xsmall: 12,
+      small: 12,
+      medium: 6,
+      large: 4,
+    }}
+    rowSpan={{xsmall: 1, small: 2, medium: 3}}
+  />
+  <Bento.Item
+    columnSpan={{
+      xsmall: 12,
+      small: 12,
+      medium: 6,
+      large: 4,
+    }}
+    rowSpan={{xsmall: 1, small: 2, medium: 3}}
+  />
+  <Bento.Item
+    columnSpan={{
+      xsmall: 12,
+      small: 12,
+      medium: 6,
+      large: 4,
+    }}
+    rowSpan={{xsmall: 1, small: 2, medium: 3}}
+  />
+  <Bento.Item
+    columnSpan={{
+      xsmall: 12,
+      small: 12,
+      medium: 6,
+      large: 4,
+    }}
+    rowSpan={{xsmall: 1, small: 2, medium: 3}}
+  />
+</Bento>
+```
+
+#### Responsive flow
+
+```jsx live
+<Bento.Item
+  flow={{
+    xsmall: 'row',
+    small: 'row',
+    medium: 'column',
+  }}
+>
+  <Bento.Content padding="normal">
+    <Bento.Heading as="h3">Heading</Bento.Heading>
+    <Text>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+      ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+      elementum.
+    </Text>
+    <Link href="#">Call to action</Link>
+  </Bento.Content>
+  <Bento.Visual position="50% 100%">
```

Comment:
Nit.. the position seems redundant here.. recommend removing to keep the example focussed.

```suggestion
  <Bento.Visual>
```